specification/servicefabric/resource-manager/Microsoft.ServiceFabric/stable/2018-02-01/cluster.json

```diff
@@ -1581,8 +1513,7 @@
       "in": "path",
       "description": "The customer subscription identifier.",
       "required": true,
-      "type": "string",
-      "x-ms-parameter-location": "method"
+      "type": "string"

```

Comment:
You need to specify a x-ms-parameter-location for global parameters. It can be client/method. But, you must have a value. Please fix it

Reply:
@sarangan12 Fixed.